components/empty-state/empty-state-illustrated.js

```diff
@@ -74,7 +74,7 @@ class EmptyStateIllustrated extends LoadingCompleteMixin(PropertyRequiredMixin(L
 		return html`
 			${this.#renderIllustration()}
 			<p class="${classMap(titleClass)}">${this.titleText}</p>
-			<p class="d2l-body-compact d2l-empty-state-description">${this.description}</p>
+			<p class="d2l-body-compact d2l-empty-state-description" tabindex="-1">${this.description}</p>
```

Comment:
I think this would yield the default browser outline, which is of course slightly different for each browser. Do we want to add a `outline: 2px solid var(--d2l-color-celestine);` for the paragraph's `focus-visible` state?

Reply:
It does use the default, yeah. I wasn't overly concerned about it, but can use the ring if we want absolute consistency.